client/my-sites/theme/index.web.js

```diff
@@ -36,6 +37,7 @@ export default function ( router ) {
 		noSite,
 		fetchThemeDetailsData,
 		details,
+		addNavigationIfLoggedIn,
```

Comment:
Adds the sidebar navigation. We'll likely want to follow up with a custom sidebar to highlight the Themes item.